src/e2e/java/teammates/e2e/cases/lnp/BaseLNPTestCase.java

```diff
@@ -234,4 +234,11 @@ protected void deleteDataFiles() throws IOException {
         Files.delete(Paths.get(pathToCsvFile));
     }
 
+    /**
+     * Sanitize the data to be CSV data.
```

Comment:
- ~data~ string, because the purpose of the string is not relevant
- CSV<b>-safe</b> ~data~ string

Reply:
Also, change to `Sanitizes`